### PR TITLE
Use native YAML

### PR DESCRIPTION
**Pull request by Fale**
##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
system/crypttab

##### ANSIBLE VERSION
```
2.2
```

##### SUMMARY
Make YAML code use the native style

### CODE REVIEW

File: system/crypttab.py [hunk 1]
```diff
@@ -74,11 +74,17 @@
 
 EXAMPLES = '''
 - name: Set the options explicitly a device which must already exist
-  crypttab: name=luks-home state=present opts=discard,cipher=aes-cbc-essiv:sha256
+  crypttab:
```
Comment (gundalow) [inline system/crypttab.py:79]: I wonder if it's worth adding a comment to remind people to quote `opts` if it contains `:`, what do you think?

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by gregdek** [comment]
Thanks @Fale. To the current maintainers, @groks please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gundalow** [comment]
needs_revision

**Comment by gregdek** [comment]
Thanks @Fale for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@Fale A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by Fale** [comment]
What do you think?

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged, thanks

**Pull request merged**